arangodb-foxx-services/src/main/services/lineage-overview.ts

```diff
@@ -28,55 +30,55 @@ const {observedWritesByRead} = require('./observed-writes-by-read');
  * It shows how far the traversal should looks for the lineage.
  * @returns za.co.absa.spline.consumer.service.model.LineageOverview
  */
-function lineageOverview(eventKey, maxDepth) {
+export function lineageOverview(eventKey: DocumentKey, maxDepth: number) {
```

Comment:
With  typescript's `export` markup, the following at the end of the file should not be needed, I assume:
```javascript
module.exports = {
    lineageOverview
}
```

Reply:
Absolutely! That's a leftover, thanks for spotting it.